pkg/runner/runner.go

```diff
@@ -222,7 +222,18 @@ func Run(cfg *Config) error {
 		},
 	)
 
+	pendingTelemetry := map[string]string{
+		"rules_enabled":    "false", // Will be written in telemetry table as `promscale_rules_enabled: false`
+		"alerting_enabled": "false", // `promscale_alerting_enabled: false`
+	}
 	if cfg.RulesCfg.ContainsRules() {
+		pendingTelemetry["rules_enabled"] = "true"
+		if cfg.RulesCfg.ContainsAlertingConfig() {
+			pendingTelemetry["alerting_enabled"] = "true"
+		} else {
+			log.Info("msg", "Alerting configuration not present in the given Prometheus configuration file. Alerting will not be initialized")
```

Comment:
I'm kinda wondering if this log line should be INFO or maybe a DEBUG line? We are so noisy as it is when starting up, we might want to hide this unless there is a problem with configuration detection.

Reply:
Yeah, I think we should change this to debug. Will push this change.